packages/ddp-server/livedata_server.js

```diff
@@ -1330,6 +1337,29 @@ Object.assign(Subscription.prototype, {
     return self._deactivated || self._session.inQueue === null;
   },
 
+  setThrottle: function (minInterval) {
+    const self = this;
```

Comment:
I know there's this `self` convention in this file (it's rather old), but I wouldn't do that for the new code.

Reply:
I've used this self mostly because of conventions + I don't particularly appreciate using ``this`` inside a `function` (the one inside setTimeout).
I will make the adjustments